frontend/lib/resolution/Resolver.cpp

```diff
@@ -1758,9 +1758,10 @@ void Resolver::validateAndSetToId(ResolvedExpression& r,
   r.setToId(id);
   if (id.isEmpty()) return;
 
-  // Validate the newly set to ID. It shouldn't refer to a module unless
-  // the node is an identifier in one of the places where module references
-  // are allowed (e.g. imports).
+  // Validate the newly set to ID.
+
```

Comment:
? I guess I can't complain cause I do this all the time 😄.

Reply:
Well, there are now two validations, so I split the comments into "overall" and "what each validation does"